operator/roles/kiali-deploy/defaults/main.yml

```diff
@@ -80,6 +80,7 @@ kiali_defaults:
       namespace: ""
       service: ""
       port: 16686
+      in_cluster_url: ""
```

Comment:
Also, I imagine that a change in the operator would trigger a change in the "users" of the operators.
In particular, if I'm not wrong, the operator populates these data from maistra data (or it's the maistra operator who creates the kiali CR).
So probably this will need to be propagated in order to be working.

Reply:
put in alphabetical order - put it above insecure_skip_verify

Reply:
> or it's the maistra operator who creates the kiali CR

Yes, this is a good point. See this template that Maistra uses when creating the Kiali CR:
 
https://github.com/Maistra/istio-operator/blob/014f116a7e2291cf7e89c4612700724a636a5ccd/helm/istio/charts/kiali/templates/kiali-cr.yaml#L65-L79

Reply:
Note also this Go code in the Maistra operator that attempts to auto-detect Jaeger and re-configured the Kiali CR:

https://github.com/Maistra/istio-operator/blob/014f116a7e2291cf7e89c4612700724a636a5ccd/helm/istio/charts/kiali/templates/kiali-cr.yaml#L65-L79

Reply:
PR sent https://github.com/Maistra/istio-operator/pull/244